tutorials/tour/_posts/2017-02-13-singleton-objects.md

```diff
@@ -10,65 +10,113 @@ num: 13
 
 next-page: xml-processing
 previous-page: pattern-matching
+assumed-knowledge: classes, methods, private-methods, packages
 ---
 
-Methods and values that aren't associated with individual instances of a [class](classes.html) belong in *singleton objects*, denoted by using the keyword `object` instead of `class`.
+Singleton objects are like classes in that they contain methods and values, but there are a few key differences.
+* There is only one instance of a singleton object so their methods and values aren't associated with individual instances of a [class](classes.html).
+* Objects have no constructors so they cannot be instantiated
```

Comment:
@som-snytt Looking back on it, I agree that it's a bit misleading. I like "a single instance of a new class" except that the word "class" could be confusing for people who assume that classes can be instantiated. I'll try to explain that. Thanks for the feedback.